src/bindings/c/tests/ov_core_test.cpp

```diff
@@ -171,33 +171,33 @@ TEST_P(ov_core, ov_core_compile_model_from_file) {
 TEST_P(ov_core, ov_core_set_property_enum) {
     auto device_name = GetParam();
     ov_core_t* core = nullptr;
-    OV_ASSERT_OK(ov_core_create(&core));
+    OV_EXPECT_OK(ov_core_create(&core));
     ASSERT_NE(nullptr, core);
 
     const char* key = ov_property_key_log_level;
     const char* mode = "WARNING";
 
-    OV_ASSERT_OK(ov_core_set_property(core, device_name.c_str(), key, mode));
+    OV_EXPECT_OK(ov_core_set_property(core, device_name.c_str(), key, mode));
     ov_core_free(core);
 }
 
 TEST_P(ov_core, ov_core_set_property_invalid_number_property_arguments) {
     auto device_name = GetParam();
     ov_core_t* core = nullptr;
-    OV_ASSERT_OK(ov_core_create(&core));
+    OV_EXPECT_OK(ov_core_create(&core));
     ASSERT_NE(nullptr, core);
 
     const char* key_1 = ov_property_key_inference_num_threads;
     const char* value_1 = "12";
-    const char* key_2 = ov_property_key_hint_performance_mode;
-    const char* value_2 = "LATENCY";
+    const char* key_2 = ov_property_key_num_streams;
+    const char* value_2 = "7";
```

Comment:
now ARM supports performance mode, does it allow to revert this line to LATENCY?

Reply:
No problem! Does CI enable arm platform test cases so that we can have a test?